src/convertToObject.js

```diff
@@ -6,7 +6,20 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const properties = sourceString.split(';');
+  const result = {};
+
+  properties.forEach((property) => {
+    const objProperty = property.trim();
+
+    const [key, value] = objProperty.split(':');
+
+    if (key && value) {
```

Comment:
The condition is correct to ensure that both key and value exist before adding them to the object. However, consider handling edge cases where key or value might contain only whitespace, which would be truthy but not a valid CSS declaration.